server/src/pages/leave-screenshots/server.js

```diff
@@ -4,7 +4,7 @@ const reactrender = require("../../reactrender");
 const { Shot } = require("../../servershot");
 const mozlog = require("../../logging").mozlog("leave-screenshots");
 
-const csrfProtection = csrf({cookie: true});
+const csrfProtection = csrf({cookie: {httpOnly: true, secure: true, sameSite: 'lax', key: '__Host-csrf'}});
```

Comment:
Sure, I can make that change. It seemed like overkill to move a one-liner into a module, but it's also not satisfying to spread this long incantation all over.